### DISCUSSION TITLE
4.3.51

### DISCUSSION THREAD

**Discussion by KillerInk** [KillerInk/FreeDcam#165] [General]
-add waveform for gles3 devices.

-add ability to change selected manual with vol+/-. 
when using a headset, you can click on center button to switch to the next manual. 
long press center to start work.

-bugfixes

<hr /><em>This discussion was created from the release <a href='https://github.com/KillerInk/FreeDcam/releases/tag/4.3.51'>4.3.51</a>.</em>

**Comment by fob014** [comment]
#1 app crash when trying to change iso from auto to manunal in front camera 
(in previou #4.3.39  version  that iso  work fine)

#2 trigger sutture settings  with volume - , +  not showing in setting

**Reply by KillerInk** [reply]
1. yes there is something wrong. wonder whats going on.
2. thats works always. select zoom or iso or something else and you can change it with vol+/-.
with plugged headset and 3 buttons(vol+/- and center) on it, you can switch to next manual with press the center. hold center to start/stop work

**Reply by fob014** [reply]
#3 .  (bug)  now it only record video for 8 sec 
while recording it does so it recording much longer but when play file it only play till 8 sec than error (null) i test many time but results was same

**Reply by KillerInk** [reply]
recording works here. need a log to see whats going on

**Reply by fob014** [reply]
> recording works here. need a log to see whats going on

how to take log?

**Reply by KillerInk** [reply]
https://github.com/KillerInk/FreeDcam/wiki/How-to-Log

**Comment by fob014** [comment]
log fil for that video recording issut


[log.txt](https://github.com/KillerInk/FreeDcam/files/6965772/log.txt)

**Reply by KillerInk** [reply]
open your 1080p profile inside the mediaprofile editor and set recording duration to 0.
then is should work. looks like your media recorder dont like it.

**Reply by fob014** [reply]
> open your 1080p profile inside the mediaprofile editor and set recording duration to 0.
> then is should work. looks like your media recorder dont like it.


now it recording works   thanks ✌️
but still some wired things happening
new video is not playable  specially in  mx player don't know why 
but Google photo app play it fine

**Reply by KillerInk** [reply]
i dont know. 
what happen on record encoding finaly, knows only the oem^^
would say its a mx player problem. cause gphoto works.

**Comment by fob014** [comment]
in video recording issue 
why sometimes it crop the  sensor  or sometime it don't even in same video profile

**Reply by KillerInk** [reply]
dont know. api2 should work. at least on all my devices. api1 no idea.

**Reply by fob014** [reply]
> dont know. api2 should work. at least on all my devices. api1 no idea.

i have that issue on api 2 
even at same video profile some time it crop  the sensor or some time it didn't

**Reply by KillerInk** [reply]
can you pls attach screenshots from your problem. cant reproduce it.

**Reply by fob014** [reply]
> can you pls attach screenshots from your problem. cant reproduce it.

in this two  screenshot you clearly see how much it crop the sensor 
& compare it  i didn't movie the camera both screenshot taken at same distance 
![Screenshot_20210902-221204](https://user-images.githubusercontent.com/80119337/131884699-d5d580d0-4422-439f-8c24-5e25fa5dca9e.png)
![Screenshot_20210902-221147](https://user-images.githubusercontent.com/80119337/131884756-2c8c6222-5c6f-4b64-9e99-985de002d6c6.png)
